zap/src/main/java/org/parosproxy/paros/extension/manualrequest/http/impl/HttpPanelSender.java

```diff
@@ -274,6 +329,14 @@ private JToggleButton getButtonUseCookies() {
         return useCookies;
     }
 
+    private JCheckBox getButtonUseCsrf() {
+        if (useCsrf == null) {
+            useCsrf = new JCheckBox();
```

Comment:
I'd suggest using a toggle button with an icon, don't know which one though :)

Reply:
Would it be okay for me to reuse one of the icons already available in the resources directory? I can pick an appropriate one and put it into my next update to this PR haha

Reply:
Absolutely, or something from the [Fugue collection](https://p.yusukekamiyamane.com/icons/search/fugue/) which we've used a bunch already.